crates/sui-core/src/db_checkpoint_handler.rs

```diff
@@ -81,6 +111,30 @@ impl DBCheckpointHandler {
         });
         sender
     }
+    async fn prune_and_compact(&self, db_path: PathBuf, epoch: u32) -> Result<()> {
+        let perpetual_db = Arc::new(AuthorityPerpetualTables::open(&db_path.join("store"), None));
+        let config = AuthorityStorePruningConfig::default();
```

Comment:
default setting would be to keep last two epochs
do we want to force aggressive pruning here?

Reply:
good point, fixed!